docs/conf.py

```diff
@@ -46,6 +47,13 @@
     "yaml",
 ]
 
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/3', None),
+    'gufe': ('https://gufe.readthedocs.io/en/latest/', None),
+    'openfe': ('https://docs.openfree.energy/en/stable/', None),
+    'perses': ('https://perses.readthedocs.io/en/latest/', None)
+}
```

Comment:
LGTM! I one nit, sort this dict by keys and add a trailing comma, so when we add to this list the diff is clean

Reply:
Awesome! Done!